### ISSUE TITLE
F1 for "Expanding Abbreviation"

### ISSUE THREAD

**Issue by nurtext** [sergeche/tea-for-coda#3]
This isn't very useful since most of the people are coding with 10 fingers on the main QWERTY layout of the keyboard. F1 requires to move the left hand slightly to the upper left corner which interrupts the workflow of coding…

Please used one of the control keys (Ctrl, Alt, Cmd) on the left in combination with any regular button on the right side of the QWERTY layout (e.g. U, I, O, P, ",", "." etc).

**Comment by sergeche** [comment]
You can change key bindings in almost any MacOS app in System Preferences > Keyboard > Keyboard Shortcuts > Application Shortcuts

**Comment by nurtext** [comment]
That's right, but not common. Think about changing the keybinding…

**Comment by sonicslp** [comment]
I completely agree, why on earth was it changed from "ctrl+," to "F1"? It makes no sense to me, the other key combination was much quicker because you don't have to move your hands as much.

**Comment by ghost** [comment]
...no development for this argument?

**Comment by sergeche** [comment]
It was fixed in v0.7 (in March, 2011): https://github.com/sergeche/zen-coding
Also, you can always change any ZC keybinding using standard Mac OS’ keyboard preferences

**Comment by ghost** [comment]
Thank you for your quick responce. I'll change keybinding.
